### PR TITLE
E-152 feat: add logout button, fix trim clientId

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: package.json [hunk 1]
```diff
@@ -54,16 +54,16 @@
     "vuex-class-modules": "^1.1.2"
   },
   "devDependencies": {
-    "@babel/core": "7.8.7",
+    "@babel/core": "7.9.0",
```
Comment (lamartire) [inline package.json:57]: Why you are updating everything in cases, when we need only one package update?

File: src/components/common/VClientId.vue [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+        class="v-client-id-input"
+      />
+      <v-button
+        is-inline
+        :disabled="disabled"
+        :is-loading="isLoading"
+        skin="ghost"
+        @click="onSet"
+      >
+        Set
```
Comment (lamartire) [inline src/components/common/VClientId.vue:17]: Locale missing

Comment (lamartire) [reply src/components/common/VClientId.vue:17]: Everywhere in this PR

File: src/components/common/VHeaderControls.vue [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  name: 'VHeaderControls',
+
+  components: {
+    VLogoutButton,
+    VClientId,
+  },
+};
+</script>
+
+<style lang="scss">
```
Comment (lamartire) [inline src/components/common/VHeaderControls.vue:22]: `scoped`

File: src/components/common/VHeaderControls.vue [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+.v-header-controls {
+  display: flex;
+}
+
+.v-header-controls-logout {
+  margin-left: auto;
```
Comment (lamartire) [inline src/components/common/VHeaderControls.vue:28]: Please, add `flex` property for all flex container children

File: src/components/common/VHeaderControls.vue [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  components: {
+    VLogoutButton,
+    VClientId,
+  },
+};
+</script>
+
+<style lang="scss">
+.v-header-controls {
+  display: flex;
```
Comment (lamartire) [inline src/components/common/VHeaderControls.vue:24]: Missing `align-items`

File: src/store/modules/ConnectModule.js [hunk 6]
```diff
@@ -25,8 +26,8 @@ class ConnectModule extends VuexModule {
   }
 
   @Action
-  async setClientId(clientId) {
-    LocalStorage.save(CLIENT_ID_KEY, clientId);
+  async setClientId(clientId = '') {
```
Comment (lamartire) [inline src/store/modules/ConnectModule.js:29]: Is it possible to remove default value? IMO we should use that only in cases when `clientId` exists

File: src/store/modules/ConnectModule.js [hunk 7]
```diff
@@ -73,17 +73,21 @@ class ConnectModule extends VuexModule {
...
 
-    const connect = new ConnectCompose({
+    this.connectInstance = new ConnectCompose({
       authUrl: ENV.VUE_APP_AUTH_URL || 'https://auth.endpass.com',
       clientId: this.clientId,
       oauthServer: ENV.VUE_APP_OAUTH_SERVER,
       widget: false,
-      plugins: [ConnectOauth, ConnectDocument, ConnectLoginButton],
+      plugins: [
+        //
```
Comment (lamartire) [inline src/store/modules/ConnectModule.js:82]: Remove

File: package.json (hunk 1 above)
Comment (dudiq) [reply package.json:57]: just up to date all deps and for no any notify in repo about security error with `old dependency` or something else

File: src/components/common/VClientId.vue (hunk 2 above)
Comment (dudiq) [reply src/components/common/VClientId.vue:17]: this repo doesn't have any locales

File: src/components/common/VHeaderControls.vue (hunk 3 above)
Comment (dudiq) [reply src/components/common/VHeaderControls.vue:22]: every component in this repo haven't scoped, just scss

File: package.json (hunk 1 above)
Comment (lamartire) [reply package.json:57]: Security errors better solve through special PR from dependabot. When you are updating dependencies directly it not exclude similar cases with security issues

Comment (lamartire) [reply package.json:57]: Then, just update ones what you need, not all packages

Comment (kazurus) [reply package.json:57]: We should use latest versions, soon as possible

**Comment by ifiri** [comment]
@dudiq logout button showing even if I not logged in. Also, logout button appears quite lower than "set" button. Maybe move it top a bit?
image

**Comment by dudiq** [comment]
> @dudiq logout button showing even if I not logged in. Also, logout button appears quite lower than "set" button. Maybe move it top a bit?
> image

we can't recognize in demo we are logged in or not.
and button is position in one line
image

**Comment by ifiri** [comment]
@kazurus approved

**Pull request merged**